src/services/firebase/Authentication.js

```diff
@@ -33,3 +33,25 @@ export function sendVerificationEmail (user) {
     .then(() => console.log('Email de verificação enviado'))
     .catch(() => console.error('Error ao enviar email de verificação'))
 }
+
+export const saveProfile = (profile) => {
+  var currentUser = Firebase.auth().currentUser
+  var pathsToUpdate = {}
+  pathsToUpdate[`profiles/${currentUser.uid}`] = profile
+
+  return new Promise((resolve, reject) => {
+    Firebase
```

Comment:
Do you think that moving this ´Firebase.database()´ to database module could be better?